PowerShell/ScubaGear/Modules/CreateReport/CreateReport.psm1

```diff
@@ -182,12 +182,26 @@ function New-Report {
     # Handle AAD-specific reporting
     if ($BaselineName -eq "aad") {
         $ReportHTML = $ReportHTML.Replace("{AADWARNING}", $AADWarning)
-        $ReportHTML = $ReportHTML.Replace("{CAPTABLES}", "")
```

Comment:
just confirming: you're removing this because it wasn't actually used to populate the CAP tables and is thus vestigial, correct?  Initially I was surprised but looking more closely it seems CAP data is populated in JS and just appended to the end of the DOM?

Reply:
Yes I removed this because when Alden was helping explain to me how the createreport file works, he realized that the CapTable wasn't actually used and was set to "" or empty both times, and was actually populated using the JS and recommended I remove it and test to make sure everything still works which it did.